### PR TITLE
Don't error when trying to put the same report to the inflight

### PR DESCRIPTION
**Pull request by mateusz-sekara**
## Solution

### CODE REVIEW

File: core/services/ocr2/plugins/ccip/ccipexec/inflight.go [hunk 1]
```diff
@@ -54,29 +53,50 @@ func (container *inflightExecReportsContainer) expire(lggr logger.Logger) {
...
+			)
+		} else {
+			tmpReports = append(tmpReports, report)
 		}
 	}
+	container.reports = tmpReports
 
 	// Otherwise not already in flight, add it.
-	lggr.Info("Inflight report added")
+	lggr.Warnw(
```
Comment (makramkd) [inline core/services/ocr2/plugins/ccip/ccipexec/inflight.go]:
```suggestion
	lggr.Infow(
```

Feels more appropriate as a log level.

File: core/services/ocr2/plugins/ccip/ccipexec/inflight.go [hunk 2]
```diff
@@ -54,29 +53,50 @@ func (container *inflightExecReportsContainer) expire(lggr logger.Logger) {
...
 	}
 	container.reports = stillInFlight
 }
 
-func (container *inflightExecReportsContainer) add(lggr logger.Logger, messages []cciptypes.EVM2EVMMessage) error {
+func (container *inflightExecReportsContainer) add(lggr logger.Logger, messages []cciptypes.EVM2EVMMessage) {
 	container.locker.Lock()
 	defer container.locker.Unlock()
 
+	tmpReports := container.reports[:0]
```
Comment (makramkd) [inline core/services/ocr2/plugins/ccip/ccipexec/inflight.go]: Is the intention here to start with an empty slice? Why not declare it that way?

File: core/services/ocr2/plugins/ccip/ccipexec/inflight.go [hunk 3]
```diff
@@ -54,29 +53,50 @@ func (container *inflightExecReportsContainer) expire(lggr logger.Logger) {
 		if time.Since(report.createdAt) > container.cacheExpiry {
 			// Happy path: inflight report was successfully transmitted onchain, we remove it from inflight and onchain state reflects inflight.
 			// Sad path: inflight report reverts onchain, we remove it from inflight, onchain state does not reflect the change so we retry.
-			lggr.Infow("Inflight report expired", "messages", report.messages)
+			lggr.Infow(
+				"Inflight report expired",
+				"minSeqNr", report.messages[0].SequenceNumber,
+				"maxSeqNr", report.messages[len(report.messages)-1].SequenceNumber,
+				"messages", report.messages,
```
Comment (makramkd) [inline core/services/ocr2/plugins/ccip/ccipexec/inflight.go]: ✔️

Comment (makramkd) [inline core/services/ocr2/plugins/ccip/ccipexec/inflight.go]: This can be pretty big right? Is this needed in the log?

**Review by makramkd — COMMENTED** [review]
LGTM overall minor comments

File: core/services/ocr2/plugins/ccip/ccipexec/inflight.go (hunk 3 above)
Comment (mateusz-sekara) [reply core/services/ocr2/plugins/ccip/ccipexec/inflight.go]: I haven't touched that, but indeed we can probably remove that

**Review by makramkd — APPROVED** [review]

**Pull request closed without merging**